.travis.yml

```diff
@@ -1,22 +1,18 @@
 language: node_js
-node_js: 6
+node_js: latest
```

Comment:
Should be stable: https://github.com/Polymer/polymer/blob/44fd1eaa6a4fb3d797155ea9c9b286f7bcca6dbb/.travis.yml#L4

Reply:
Oh, your commit and my comment passed each other in the timespace :smile: